README.rst

```diff
@@ -19,9 +19,13 @@ Functionality
    expressions after the entire file has been read.  This enables support for
    first-class forward references that stub files use.
 
-The latter should ideally be merged into ``flake8`` as the integration is
-currently pretty brittle (might break with future versions of ``pyflakes``,
-``flake8``, or due to interactions with other overly clever plugins).
+3. Provides a number of ``.pyi``-specific warnings that enforce typeshed's
+   style guide.
+
+Be careful when using this plugin in the same environment as other flake8
+plugins, because they might generate errors that are not appropriate for
+``.pyi`` files (e.g., about missing docstrings). We recommend running
+``flake8-pyi`` in a dedicated virtual environment in your CI.
```

Comment:
It's uncommon to use virtualenvs in github actions, because each job has a fresh environment anyway:

```suggestion
``flake8-pyi`` in a dedicated environment in your CI.
```